README.md

```diff
@@ -1041,6 +1041,8 @@ Table of Contents
   * [Qoddi](https://qoddi.com) - PaaS service similar to Heroku with a developer-centric approach and all inclusive features. Free tier for static assets, staging and developer apps.
   * [FreeFlarum](https://freeflarum.com/) - Community powered free Flarum hosting for up to 250 users (donate to remove watermark from footer).
   * [fleek.co](https://fleek.co/) - Build modern sites and apps on the Open Web and its protocols seamlessly free for, unlimited websites and 50 GB/month bandwidth.
+  * [000webhost](https://000webhost.com) - Free PHP, MySQL, and cPanel Hosting.
+  * [InfinityFree](https://infinityfree.com) - Completely free website hosting with PHP, MySQL and 99% uptime.
```

Comment:
We're not adding these kinds of sites anymore, see https://github.com/ripienaar/free-for-dev/pull/2437